samples/net/dsa/src/main.c

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2020 DENX Software Engineering GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(net_dsa_sample, LOG_LEVEL_DBG);
+
+#include <zephyr.h>
+
+#include <zephyr.h>
+#include <errno.h>
+
+#include <net/net_core.h>
+#include <net/net_l2.h>
+#include <net/net_if.h>
+#include <net/ethernet.h>
+#include <net/dsa.h>
+
+/* User data for the interface callback */
+struct ud {
```

Comment:
have a more self explanatory name, at least to differentiate from the struct name to the actual variable name.

Reply:
This is the "user data" - I've followed the LLDP example with naming.
I've updated the code to have more explanatory name.